packages/elements-core/src/__fixtures__/operations/urlencoded-post-oneof.ts

```diff
@@ -0,0 +1,80 @@
+import { IHttpOperation } from '@stoplight/types';
+
+export const httpOperation: IHttpOperation = {
+  id: '?http-operation-id?',
+  iid: 'POST_todos',
+  method: 'post',
+  path: '/todos',
+  summary: 'Create Todo',
+  responses: [
+    {
+      id: '?http-response-200?',
+      code: '200',
+    },
+  ],
+  servers: [
+    {
+      id: '?http-server-todos.stoplight.io?',
+      url: 'https://todos.stoplight.io',
+    },
+  ],
+  request: {
+    body: {
+      id: '?http-request-body?',
+      contents: [
+        {
+          id: '493afac014fa8',
+          mediaType: 'application/x-www-form-urlencoded',
+          examples: [
+            {
+              id: '83adf9aba208e',
+              key: 'Request New Token',
+              value: {
+                username: 'Neovest provided username',
+                password: 'corresponding password',
+              },
+            },
+            {
+              id: '9dcf9cf7d39ea',
+              key: 'Refresh Token',
+              value: {
+                refresh_token: 'some string',
+              },
+            },
+          ],
+          encodings: [],
+          schema: {
+            oneOf: [
```

Comment:
This line is the important difference between this example and `packages/elements-core/src/__fixtures__/operations/urlencoded-post.ts`, the use of `oneOf` at the top level of the schema.